instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/TracingExecutionInterceptor.java

```diff
@@ -233,6 +231,25 @@ public SdkRequest modifyRequest(
     return request;
   }
 
+  @Override
+  public void afterMarshalling(
```

Comment:
As far as I know scope is used to suppress nested http client spans. It is possible that setting up the scope in some other (later) phase would also work.